### ISSUE TITLE
Extracting data from reachSet

### ISSUE THREAD

**Issue by akaph2p** [TUMcps/CORA#48]
Hi,

I am trying to extract the YData from the reachSet after using reach. I specifically want to do this to add a constant offset to each y-value of the reach set and then plot Y vs time. Is there a method to extract that information from the zonotope object? 

This is my code:
```
R = reach(reactor_ss, params, options);
```

The output is R which is a 1x1 reachSet consisting of timePoint, timeInterval, parent. and loc. The 'timePoint' is a structure which contains the set, time, and error. 'set' is a cell array containing zonotopes for each time point of the reachset. Is there a away to extract the 'set' information to manipulate and plot it. 

![reachset](https://github.com/TUMcps/CORA/assets/72284197/1679684b-3aa0-4715-a0a0-baf2afd71b8d)

**Comment by TUMcps** [comment]
Dear akaph2p,

Thanks for reaching out.

CORA allows a direct manipulation of sets, including the `reachSet` object:

```
% ....
R = reach(reactor_ss, params, options);

figure; hold on;
dim = 2;
plotOverTime(R, dim);
plotOverTime(R+1, dim); % shifts all dimensions
plotOverTime(R+[0;1;0;0;0;0], dim) % only shifts to be plotted dimension 2
```

See also `reachSet/plus` and `reachSet/mtimes`. 

If you want to manipulate the sets in other ways, you can also extract the computed zonotopes from the `reachSet` object:

```
% ...
R = reach(reactor_ss, params, options);

Z1 = R.timeInterval.set{1};
Z2 = R.timeInterval.set{2};
``` 

Hope that helps!

**Comment by akaph2p** [comment]
Thank you, this is very helpful

**Comment by akaph2p** [comment]
Hi,

Can i do something similar with the simRes function? I tried to extract each cell from simRes but that becomes cumbersome when I change the number of simOpt.points. 

```
simRes = simulateRandom(reactor_ss, params, simOpt);

% Adding back the offsets
R = R+[n_0;C1_0;Trx_0;0;0;mdotp_0];

simResnew(:,1) = simRes.x{1,1}(:,1) + n_0;
simResnew(:,2) = simRes.x{1,1}(:,2) + C1_0;
simResnew(:,3) = simRes.x{1,1}(:,3) + Trx_0;
simResnew(:,4) = simRes.x{1,1}(:,4);
simResnew(:,5) = simRes.x{1,1}(:,5);
simResnew(:,6) = simRes.x{1,1}(:,6) + mdotp_0;
simRestime = simRes.t{1,1};
```

**Comment by toladnertum** [comment]
Unfortunately, this is not yet possible but I have to agree that this manipulation is a bit cumbersome.

I'll implement the required operations next week, and they will then be available with the next release (v2024.1.2).

**Comment by akaph2p** [comment]
That would be great. Thank you for your prompt response.

**Comment by toladnertum** [comment]
We just released CORA v2024.1.2.
You can now update the `simRes` object as follows:

```
simRes = simRes + [n_0; C1_0; Trx_0; 0; 0; mdotp_0];
```

I hope this solves your issue.
